lib/ResolverFactory.js

```diff
@@ -294,6 +297,7 @@ function createOptions(options) {
 		preferRelative: options.preferRelative || false,
 		preferAbsolute: options.preferAbsolute || false,
 		restrictions: new Set(options.restrictions),
+		tsconfig: typeof options.tsconfig === "undefined" ? true : options.tsconfig,
```

Comment:
By default it should be false, not the each project uses ts, so let's avoid extra job, developer should set `resolve.tsconfig: true | pathToTsconfig.json` to enable this plugin